src/main/java/com/maybe/maybe/entity/Desk.java

```diff
@@ -1,13 +1,16 @@
 package com.maybe.maybe.entity;
 
 import javax.persistence.Column;
-import javax.persistence.MappedSuperclass;
+import javax.persistence.Entity;
+import javax.persistence.Table;
 import javax.validation.constraints.NotNull;
 
-@MappedSuperclass
-public abstract class AbstractNameEntity extends AbstractEntity {
+
+@Entity
+@Table(name = "desk")
+public class Desk extends AbstractEntity {
     @NotNull
-    @Column(name = "name",nullable = false)
+    @Column(name = "name", nullable = false)
     private String name;
 
     public String getName() {
```

Comment:
I think you have to add a field DeskState deskState in this entity and map it with enum DeskState